Widgets/WidgetBase/VarmintWidgetRendering.cs

```diff
@@ -33,30 +33,32 @@ public void AdvanceAnimations(GameTime gameTime)
         //--------------------------------------------------------------------------------------
         public virtual void RenderMe(GameTime gameTime)
         {
-            if(Renderer == null)
+            if (Renderer == null)
             {
-                throw new InvalidOperationException("The Renderer property is null on " + this.GetType().Name + " with Name=" + Name);
+                // throw new InvalidOperationException("The Renderer property is null on " + this.GetType().Name + " with Name=" + Name);
```

Comment:
You can remove this comment now that I have fixed that code that was causing the crash